docker-compose.yml

```diff
@@ -17,6 +17,7 @@ services:
       dockerfile: ./frontend/Dockerfile
     volumes:
       - ./frontend/pages:/usr/src/app/pages
+      - ./frontend/components:/usr/src/app/components
```

Comment:
does this need to be in this pr?

Reply:
this line is for code reloading in Docker, and goes with the component folder, which I'm adding in this pr -- so I think yes